File: languages/go/utils/utilities.go
package utils

import (
	"fmt"
	"math"
)

func RoundToDecimals(value float64, precision int) float64 {
	factor := math.Pow(10, float64(precision)) // Calculate 10 ^ precision
	return math.Round(value*factor) / factor   // Round and divide
}

func Fatal(s string) {
	panic(s)
}

func Fatalf(s string, a ...any) {
	panic(fmt.Sprintf(s, a...))
}

File: languages/nodejs/src/utils/utilities.ts
/**
 * Rounds value based on the given precision.
 * @param value Value being rounded.
 * @param precision The number of decimals to round value to.
 * @returns Rounded value.
 */
export function roundToDecimals(value: number, precision: number) {
    const factor = Math.pow(10, precision);
    return Math.round(value * factor) / factor;
}